### PR TITLE
install ceph-common pkg on origin to support rbd provisioning

### PR DESCRIPTION
**Pull request by rootfs**
@childsb @sdodson

### CODE REVIEW

**Comment by rootfs** [comment]
[test]

File: images/base/Dockerfile [hunk 1]
```diff
@@ -7,7 +7,7 @@
 FROM centos:centos7
 
 RUN INSTALL_PKGS="which git tar wget hostname sysvinit-tools util-linux bsdtar epel-release \
-      socat ethtool device-mapper iptables tree findutils nmap-ncat e2fsprogs xfsprogs lsof device-mapper-persistent-data" && \
+      socat ethtool device-mapper iptables tree findutils nmap-ncat e2fsprogs xfsprogs lsof device-mapper-persistent-data ceph-comon" && \
```
Comment (sdodson) [inline images/base/Dockerfile]: ceph-common

Comment (rootfs) [inline images/base/Dockerfile]: nice!

**Comment by rootfs** [comment]
[test]

**Comment by rootfs** [comment]
@childsb @sdodson can you retest it? thanks

**Comment by sdodson** [comment]
[test]

**Comment by rootfs** [comment]
@eparis @childsb @sdodson the tests failure look flaky to me. Can you help? Thanks.

**Comment by sdodson** [comment]
flake https://github.com/openshift/origin/issues/12905
[test

**Comment by sdodson** [comment]
flake https://github.com/openshift/origin/issues/12905
[test]

**Comment by sdodson** [comment]
flake https://github.com/openshift/origin/issues/12923

**Comment by sdodson** [comment]
[merge]

**Comment by sdodson** [comment]
@rootfs needs to be picked into release-1.5 now that the branch was created

**Comment by eparis** [comment]
[merge]

**Comment by eparis** [comment]
[merge] because you are flaky crap...

**Comment by eparis** [comment]
[merge] @smarterclayton how to I get free of these?

**Comment by smarterclayton** [comment]
We've got a couple of correlated problems.  Ensure the tests have green
runs and I'll be doing selective batch merging this evening.

On Feb 22, 2017, at 3:30 PM, Eric Paris <notifications@github.com> wrote:

[merge] @smarterclayton <https://github.com/smarterclayton> how to I get
free of these?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/12896#issuecomment-281793313>,
or mute
the thread

**Comment by smarterclayton** [comment]
Flake #11016 and https://github.com/openshift/origin/issues/13067

[merge]

**Pull request merged**